src/sentry/search/events/fields.py

```diff
@@ -2178,6 +2178,42 @@ def __init__(self, dataset: Dataset, params: ParamsType):
                     ),
                     default_result_type="integer",
                 ),
+                SnQLFunction(
+                    "count",
+                    snql_aggregate=lambda _, alias: Function(
+                        "count",
+                        [],
+                        alias,
+                    ),
+                    default_result_type="integer",
+                ),
+                SnQLFunction(
+                    "last_seen",
+                    snql_aggregate=lambda _, alias: Function(
+                        "max",
+                        [self.column("timestamp")],
+                        alias,
+                    ),
+                    default_result_type="integer",
```

Comment:
does this need to be date type?
```suggestion
                    default_result_type="date",
```